GuildedRoseLLCSpec/DetailsViewControllerSpec.swift

```diff
@@ -9,14 +9,62 @@ class DetailsViewControllerSpec: QuickSpec {
         describe("View did load"){
             it("sets the item using the setItem closure"){
                 let detailsController = DetailViewController()
+                detailsController.itemName = UILabel()
+                detailsController.quality = UILabel()
+                detailsController.sellIn = UILabel()
+                
                 detailsController.setItem = {
                     () in return Item(name: "FooBar")
```

Comment:
The way this is formatted makes it a bit hard to read. Typically, the `in` is on the same line as the opening curly brace.

I would recommend looking up and following a style guide.

Reply:
Also, I'm unsure as to whether or not `()` before the `in` is conventional, so I would double check that detail too.

Reply:
changed